src/App.js

```diff
@@ -1,29 +1,19 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+import Calculator from './component/Calculator';
+
+class App extends React.Component {
+  constructor(props) {
```

Comment:
- [ ] Kindly make sure the `App.js` component is a **functional component** and not a `class` component as required for the project to `Use functional components` . i.e.
```JavaScript
...
function App() {
  return <div>{/* Calculator app design goes here */}</div>
}

// or

const App = () => {
  return <div>{/* Calculator app design goes here */}</div>
}
```